packages/near-fast-auth-signer/src/lib/firestoreController.ts

```diff
@@ -47,12 +48,23 @@ class FirestoreController {
     fakPublicKey,
     lakPublicKey,
     gateway,
+    accountId
+  }: {
+    fakPublicKey: string;
+    lakPublicKey: string;
+    gateway: string;
+    accountId?: string;
   }) {
     const parser = new UAParser();
     const device = parser.getDevice();
     const os = parser.getOS();
     const browser = parser.getBrowser();
     const dateTime = new Date().toISOString();
+
+    if (accountId && fakPublicKey) {
+      await window.firestoreController.addAccountIdPublicKey(fakPublicKey, accountId);
```

Comment:
Why not set this with the `device` and other metadata being set below? If it's for reusability in cases where we only set `accountId` we should chain the promise so that the same error handling applies

Reply:
Good point, I will change it

Reply:
Updated :)